### ISSUE TITLE
KeyError: 'histogram'

### ISSUE THREAD

**Issue by Anonymous941** [bamos/zsh-history-analysis#8]
When running `./analyze.py topCommands`, this error is shown:

```plain
y: Command, x: Frequency
Traceback (most recent call last):
  File "/.../zsh-history-analysis/./analyze.py", line 192, in <module>
    chart(colors=[], data=data, args=chart_args, labels=labels)
  File "/.../.venv/lib/python3.12/site-packages/termgraph/termgraph.py", line 534, in chart
    if args["histogram"]:
       ~~~~^^^^^^^^^^^^^
KeyError: 'histogram'
```

**Comment by Anonymous941** [comment]
I fixed it and made a PR in #9

**Comment by bamos** [comment]
thanks!
